src/winforms/toga_winforms/widgets/textinput.py

```diff
@@ -56,5 +56,5 @@ def winforms_text_changed(self, sender, event):
         if self.interface._on_change:
             self.interface.on_change(self.interface)
 
-    def winforms_click(self, sender, event):
+    def winforms_select_all_text(self, sender, event):
```

Comment:
I have recently reviewed the Winforms backend to check for consistency in naming of the native event handlers: the convention is to name the handler `winforms_<winforms event name in snake case>`, so it should be `winforms_double_click`.

Reply:
Done :)